Documentation/dev/libvirt-howto.md

```diff
@@ -214,11 +217,73 @@ master0# export KUBECONFIG=/var/opt/tectonic/auth/kubeconfig
 master0# kubectl get -n tectonic-system pods
 ```
 
-## Connect to the cluster console
+### Connect to the cluster console
 This will take ~30 minutes to be available. Simply go to `https://${CLUSTER_NAME}-api.${BASE_DOMAIN}:6443/console/` (e.g. `test1.tt.testing`) and log in using the credentials above.
 
 
-# Libvirt vs. AWS
-1. There isn't a load balancer. This means:
+
+## FAQ
+
+### Libvirt vs. AWS
+1. There isn't a load balancer on libvirt. This means:
     1. We need to manually remap ports that the loadbalancer would
```

Comment:
This isn't true after https://github.com/openshift/installer/pull/242.

Reply:
Due to this change? https://github.com/openshift/installer/pull/242/files#diff-2220e4781c4aed368880185bd3f81561L83

Reply:
I'm not sure what you're referring to here, but I'm also not sure what the intention of that statement is. What would be interesting to say here is how to test load balancing of the API in case of multiple master nodes. Not sure if we currently (want to) support this though. I'm fine removing this statement and revisit later.